ci/Jenkinsfile.tests-e2e

```diff
@@ -97,44 +94,27 @@ pipeline {
     }
 
     stage('Client') {
-      environment {
-        STATUS_RUNTIME_GANACHE_NETWORK_RPC_URL = "http://localhost:${env.GANACHE_RPC_PORT}"
-      }
       steps { script {
         linux.bundle('nim_status_client')
       } }
     }
 
-    stage('Containers') {
-      parallel {
-        stage('Ganache') { steps { script {
-          ganache = docker.image(
-            'trufflesuite/ganache:v7.4.1'
-          ).run(
-            ["-p 127.0.0.1:${env.GANACHE_RPC_PORT}:8545",
-             "-v ${env.WORKSPACE}/test/ui-test/fixtures/ganache-dbs/goerli:/goerli-db"].join(' '),
```

Comment:
@anastasiyaig do the testing team plans to migrate the text fixture to the new tool or we should remove the ganache DB and data folder also?

Reply:
we want to try other solution, but ganache is dead completely so lets remove everything